python/openai/openai_frontend/engine/utils/triton.py

```diff
@@ -36,14 +36,34 @@ def _create_vllm_inference_request(
     model, prompt, request: CreateChatCompletionRequest | CreateCompletionRequest
 ):
     inputs = {}
-    excludes = {"model", "stream", "messages", "prompt", "echo"}
+    # Exclude non-sampling parameters so they aren't passed to vLLM
```

Comment:
NOTE: May make more sense to explicitly "include" support sampling parameters, but will be of a similar length or longer. Both approaches likely require periodic updates to either include a new sampling field, or exclude a non-sampling field.